pybandits/actions_manager.py

```
@@ -949,11 +958,23 @@ def _maybe_crawl_model(model: CmabModelType):
 
+    @staticmethod
+    def _get_expected_context_size(model: CmabModelType):
+        """
+        Utility function to get the expected context size for the model.
+        """
+        if isinstance(model, BaseBayesianNeuralNetworkMO):
+            return model.models[0].input_dim
+        elif isinstance(model, (BaseBayesianNeuralNetwork, BaseQuantitativeBayesianNeuralNetwork)):
+            return model.input_dim
+        else:
+            raise ValueError(f"Unsupported model type: {type(model)}")
```

Ruff (0.15.2)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)